frontend/src/features/Dashboard/components/DashboardForm/VigilanceAreas/Panel/RegulatoryAreas.tsx

```diff
@@ -0,0 +1,152 @@
+    event.stopPropagation()
+
+    if (!regulatoryArea?.id) {
+      return
+    }
+    if (regulatoryIdsToBeDisplayed?.includes(regulatoryArea.id)) {
+      dispatch(dashboardActions.removeRegulatoryIdToBeDisplayed(regulatoryArea.id))
+    } else {
+      dispatch(dashboardActions.addRegulatoryIdToBeDisplayed(regulatoryArea.id))
+      if (!regulatoryArea?.bbox) {
+        return
+      }
+      const extent = transformExtent(
+        regulatoryArea?.bbox,
+        new Projection({ code: WSG84_PROJECTION }),
+        new Projection({ code: OPENLAYERS_PROJECTION })
+      )
+      dispatch(setFitToExtent(extent))
+    }
+  }
+
+  return (
+    <>
+      {regulatoryAreaId && <StyledRegulatoryAreasPanel layerId={regulatoryAreaId} onClose={closeRegulatoryAreapanel} />}
+      <PanelSubPart>
+        <PanelInlineItemLabel>Réglementations en lien</PanelInlineItemLabel>
+        {regulatoryAreas &&
+          regulatoryAreas.length > 0 &&
+          regulatoryAreas.map(regulatoryArea => (
+            <RegulatoryAreaContainer key={regulatoryArea?.id}>
+              <RegulatoryAreaName>
+                <LayerLegend
+                  layerType={MonitorEnvLayers.REGULATORY_ENV}
+                  legendKey={regulatoryArea?.entity_name ?? 'aucun'}
+                  type={regulatoryArea?.thematique ?? 'aucun'}
+                />
+                <span title={regulatoryArea?.entity_name}>{regulatoryArea?.entity_name}</span>
+              </RegulatoryAreaName>
+
+              <ButtonsContainer>
+                <StyledButton
+                  accent={Accent.TERTIARY}
+                  color={regulatoryAreaId === regulatoryArea?.id ? THEME.color.charcoal : THEME.color.lightGray}
+                  Icon={Icon.Summary}
+                  onClick={e => onClickRegulatoryZoneMetadata(e, regulatoryArea?.id)}
+                  title={
+                    regulatoryAreaId === regulatoryArea?.id
+                      ? 'Fermer la zone réglementaire'
+                      : 'Afficher la réglementaire'
+                  }
+                />
+
+                <StyledButton
+                  accent={Accent.TERTIARY}
+                  color={
+                    regulatoryArea?.id && regulatoryIdsToBeDisplayed?.includes(regulatoryArea?.id)
+                      ? THEME.color.charcoal
+                      : THEME.color.lightGray
+                  }
+                  Icon={Icon.Display}
+                  onClick={e => showRegulatoryAreaLayer(e, regulatoryArea)}
+                  title={
+                    regulatoryArea?.id && regulatoryIdsToBeDisplayed?.includes(regulatoryArea?.id)
+                      ? 'Cacher la zone'
+                      : 'Afficher la zone'
+                  }
+                />
+              </ButtonsContainer>
+            </RegulatoryAreaContainer>
+          ))}
+      </PanelSubPart>
+    </>
+  )
+}
+const RegulatoryAreaContainer = styled.div`
```

Comment:
nitpick: on peut mettre un span au lieu d'une div car c'est du contenu textuelle (meme si il y a la légende comprise je pense )
```suggestion
const RegulatoryAreaContainer = styled.div`
```